src/main/java/frc/robot/RobotContainer.java

```diff
@@ -199,6 +190,16 @@ public void smartDashSetup() {
                                         "Dynamic Forward", CharacterizationRoutine.DYNAMIC_FORWARD, "Dynamic Backward",
                                         CharacterizationRoutine.DYNAMIC_BACKWARD))),
                 autoFactory::getCharacterizationRoutine);
+
+        LoggedNetworkString autoInput = new LoggedNetworkString("SmartDashboard/AutoPipes", "");
+
+        autoChooser.addRoutine("L2 Auto", List.of(
+                new AutoQuestion<>("Starting Postion",
+                        Map.of("Left Left", StartingPosition.START_LL, "Left Center", StartingPosition.START_LC,
+                                "Left Right", StartingPosition.START_LR, "Right Left", StartingPosition.START_RL,
+                                "Right Center", StartingPosition.START_RC, "Right Right", StartingPosition.START_RR)),
```

Comment:
Again, this may be unclear to whoever is setting up the driver station. Maybe saying like "Left of Blue Barge Zone" or something is clearer.